textproc/typst/Portfile

```diff
@@ -4,9 +4,9 @@ PortSystem          1.0
 PortGroup           cargo   1.0
 PortGroup           github  1.0
 
-github.setup        typst typst 0.12.0 v
+github.setup        typst typst 0.13.0 v
 # Change github.tarball_from to 'releases' or 'archive' next update
-github.tarball_from tarball
+github.tarball_from releases
```

Comment:
are you sure upstream provides "releases"? I don't think this will actually do anything since "fetch.type git" is set; I am not convinced that is required. @herbygillot

Reply:
what I meant to say is that ideally we do not want to use "fetch.type git". So try removing that line and then use "github.tarball_from archive".

Reply:
This is correct; `github.tarball_from` will have no effect because the `fetch.type` is `git`.

We would need to remove `fetch.type git` and add a `checksums` block.